pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools >= 77.0.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pals-schema"
+version = "25.08"
+dependencies = [
+  "pydantic",
+  "pytest",
+  "pyyaml",
+  "toml",
+]
+requires-python = ">=3.9"
+authors = [
+  { name="Axel Huebl", email="axelhuebl@lbl.gov" },
+  { name="Chad Mitchell", email="ChadMitchell@lbl.gov" },
+  { name="David Carl Sagan", email="david.sagan@cornell.edu" },
+  { name="Jean-Luc Vay", email="jlvay@lbl.gov" },
+  { name="Edoardo Zoni", email="ezoni@lbl.gov" },
+]
+maintainers = [
+  { name="Axel Huebl", email="axelhuebl@lbl.gov" },
+  { name="Edoardo Zoni", email="ezoni@lbl.gov" },
+]
+description = "Python implementation for the Particle Accelerator Lattice Standard (PALS)"
+readme = "README.md"
+license = "CC-BY-4.0"
+license-files = ["LICEN[CS]E*"]
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "Operating System :: OS Independent",
```

Comment:
Feel free to take a few more classifiers from https://github.com/BLAST-ImpactX/impactx/blob/11748f5ba2389798910cf8303db49f9627929bd3/setup.py#L273-L294